### ISSUE TITLE
Content populated in entries field should respect source/section setting 

### ISSUE THREAD

**Issue by magnusskogen** [studioespresso/craft3-seeder#23]
If I limit an entries field to a specific source (say "News"), Seeder does not respect this setting — it adds entries from all sections. Only tested this inside a Matrix, not sure if that has anything to do with it.

**Comment by janhenckens** [comment]
@magnusskogen could you post the full command you're using? Thanks!

**Comment by magnusskogen** [comment]
Sure! You mean this?

`./craft seeder/generate/entries --section=project --count=10`

**Comment by janhenckens** [comment]
Yes! So your section handle would be `project` in this case?

**Comment by magnusskogen** [comment]
Yes! Just to clarify my specific case: I have a Matrix, where one of the fields is an entries-field where the user can only select entries from the section "News". Seeder adds entries from all sections, not only "News".

**Comment by Bcdo** [comment]
To clarify further. If you have an entries field in general, that restrics what you can choose to relate with, it will disregard this restriction. So if your entries field has the restriction of only choosing entries to relate to in the channel of "news", the seeding will still be able to choose of all the other sections that are created, not only the entries in the "news" section.
